tracer/src/Datadog.Trace/Debugger/Snapshots/SnapshotSerializerFieldsAndPropsSelector.cs

```diff
@@ -49,7 +49,7 @@ internal static SnapshotSerializerFieldsAndPropsSelector CreateDeepClonerFieldsA
         /// <param name="type">Object type</param>
         /// <param name="cts">Cancellation token source</param>
         /// <returns>Collection of fields and auto properties</returns>
-        private static IEnumerable<FieldInfo> GetAllFields(Type type, CancellationTokenSource cts)
+        private static IEnumerable<MemberInfo> GetAllFields(Type type, CancellationTokenSource cts)
```

Comment:
I'm not super clear on why we need this change?

Reply:
Fixed in 963a0d8f0b02952b982c82fd16239363573c7b19